src/osha/oira/client/browser/client.py

```diff
@@ -229,8 +229,11 @@ def results(self):
             .filter(Account.loginname.in_(recipients))
             .distinct()
         )
+        country = path.partition("/")[0]
+        # XXX Language should come from user preferences in the future
+        language = country
         return {
-            account: path.partition("/")[0]
+            account: {"country": country, "language": language}
             for account, path in query
             if path.partition("/")[0] != "eu"
```

Comment:
```suggestion
```
I think this should go away because you want to know that the user is a `eu` user, right?